test/topology_custom/test_hints.py

```diff
@@ -49,3 +50,61 @@ def get_hints_written_count(server):
     # Verify hints are written
     hints_after = get_hints_written_count(servers[0])
     assert hints_after > hints_before
+
+@pytest.mark.asyncio
+async def test_sync_point(manager: ManagerClient):
```

Comment:
Please run this test 100 times on your machine. I remember that the base64 encoding of the sync points can sometimes have special characters which `curl` didn't like. I know that you are using the `requests` library here instead, but I'm just a little bit paranoid, but please soothe my anxiety and let's see if `requests` escapes everything properly and we don't introduce a flaky test by mistake :)

Reply:
100 consecutive runs on dev have succeeded